docs/source/authentication_with_modelon_impact.rst

```diff
@@ -24,63 +18,23 @@ in the JupyterHub environment:
     >>>> client = Client()
 
 
-JupyterHub authentication
-*************************
+Authentication outside JupyterHub environment
+*********************************************
 

```

Comment:
I suggest merging with the previous section and stop mentioning JupyterHub which is a pure internal component now. We can still be talking "from inside Modelon Impact (either custom functions, scripts or notebooks) vs from outside.